build_badtrack.py

```diff
@@ -30,7 +30,7 @@
 control_content = """\
 Package: badtrack
 Version: 1.0.0
```

Comment:
can u also test if you just have version 1? without any decimals. I want to version based on git commit number and commit id. So it will instead look like:
```
1-abf3774e
```
instead of
```
1.0.0
```
Just test, i dont expect a commit for it